modules/ding_event/ding_event.module

```diff
@@ -336,6 +348,7 @@ function ding_event_form_views_exposed_form_alter(&$form, &$form_state) {
     if (!empty($form['title'])) {
       $form['title']['#attributes']['placeholder'] = t('Search for an event');
     }
+    $form['title']['#autocomplete_path'] = 'culture_events/date/autocomplete';
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-937

Reply:
This issue does not describe this change. I don't know what this does. However "culture_events" is hardcoded - that seems wrong.

Reply:
https://github.com/easySuite/ding2/commit/47f30b04ea21c0b4a16ea56fd63694542989792a

Reply:
And eventually, the implementation resides in https://github.com/easySuite/culture_frontend/blob/develop/culture_frontend.module#L24

Reply:
In which field? We already have autocomplete of all CTs in easyOPAC.

Reply:
This should not be ported.